static/src/js/components/FailedToLoadCollectionsMessage/FailedToLoadCollectionsMessage.scss

```diff
@@ -0,0 +1,28 @@
+@use '../../../css/vendor/bootstrap/configuration' as bootstrap;
+@use '../../../css/utils';
+
+.failed-to-load-collections {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  &__message-text {
+    flex: 1;
+    align-self: center;
+  }
+
+  &__see-alerts {
+    margin-left: 1rem;
+
+    .button__icon {
+      border: 1px solid utils.$color__spacesuit-white;
+      border-radius: 50%;
+      padding: 0.2rem;
+      width: 1.25rem;
+      height: 1.25rem;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+}
```

Comment:
nit; newline. Also you should be able to use classnames from bootstrap for these